hazelcast-jca/src/main/java/com/hazelcast/jca/ResourceAdapterImpl.java

```diff
@@ -157,6 +191,22 @@ public void setConfigurationLocation(String configurationLocation) {
         this.configurationLocation = configurationLocation;
     }
 
+    /**
+     * @return True if this adapter has been configured to work with a Hazelcast Client Instance. False, otherwise
+     */
+    public Boolean isClient() {
+        return client;
+    }
+
+    /**
+     * Called by the container
+     *
+     * @param client If this is adapter is to use a Hazelcast Client Instance
```

Comment:
Can we improve this javadoc?
This description could be an option. `sets whether client mode is enabled`